app/src/main/java/io/homeassistant/companion/android/vehicle/MainVehicleScreen.kt

```diff
@@ -326,4 +277,69 @@ class MainVehicleScreen(
             }
         }
     }
+
+    fun addDomainList(domains: MutableSet<String>): ItemList.Builder {
```

Comment:
Do `addDomainList` and `nativeModeActionStrip` need to live in here? If not, it would make more sense to move it to a common UI elements class or something.

Reply:
`addDomainList` yes because we need to push a screen which contains a lot of variables. `nativeActionStrip` no we can move that out

Reply:
Moved over action strip, ended up keeping addDomainList due to the amount of variables